src/Microsoft.OData.Core/ODataErrorDetail.cs

```diff
@@ -22,5 +24,16 @@ public sealed class ODataErrorDetail
         /// <summary>Gets or sets the target of the particular error.</summary>
         /// <returns>For example, the name of the property in error</returns>
         public string Target { get; set; }
+
+        /// <summary>
+        /// Serilization to Json format string.
+        /// </summary>
+        /// <returns>The string in Json format</returns>
+        public override string ToString()

```

Comment:
Here is what is specify in https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Core/ODataError.cs#L44:
        /// <summary>
        /// A collection of JSON objects that MUST contain name/value pairs for code and message, and MAY contain
        /// a name/value pair for target, as described above.
        /// </summary>
        /// <returns>The error details.</returns>
        public ICollection<ODataErrorDetail> Details { get; set; }